terrawrap/models/wrapper_config.py

```diff
@@ -43,11 +43,13 @@ def __init__(
         region: str = None,
         dynamodb_table: str = None,
         role_arn: str = None,
+        use_lockfile: Optional[bool] = None,
```

Comment:
Should the default be false? Otherwise this will be eventually evaluated as `use_lockfile=none`